styles.css

```diff
@@ -5,8 +5,60 @@ body {
   margin: 0;
 }
 
+.mobile-modal {
+  background-color: hsla(234, 100%, 50%, 0.595);
+  display: flex;
+  flex-direction: column;
+  padding: 5% 40px 20px 28px;
+  position: absolute;
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 0;
+  z-index: 3;
+  height: 100%;
+  width: 100%;
+}
```

Comment:
- [ ] It will be better to make the model `position:fixed;` ,so when scroll the menu it will be fixed in the screen.
- [ ] Also add `margin-top:0` because there is some space in the top of modal to fix it.
- [ ] Consider to add background blur effect like the figma design.

Reply:
Okay thank you for your feed-back